packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -9,19 +9,22 @@ import { Icon, IIconStyles, IStackStyles, ITheme, memoizeFunction, Stack, useThe
  */
 export type NotificationIconProps = {
   chatMessagesCount: number;
+  label?: string;
 };
 /**
  * @private
  */
 export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
-  const { chatMessagesCount } = props;
+  const { chatMessagesCount, label } = props;
   const theme = useTheme();
   const renderNumber = (numberOfMessages): JSX.Element => {
     if (numberOfMessages < 1) {
       return <></>;
     } else {
       return (
-        <Stack styles={notificationTextStyles(theme)}>{numberOfMessages < 9 ? <>{numberOfMessages}</> : <>9+</>}</Stack>
+        <Stack aria-label={label} role={'alert'} styles={notificationTextStyles(theme)}>
+          {numberOfMessages < 9 ? <>{numberOfMessages}</> : <>9+</>}
```

Comment:
Nit: consider wrapping in a `<Text>` and applying the `role={status}`

Reply:
(not sure if this needs a seperate aria label also to say this is the number of unread messages?)

Reply:
I can ask Alex Pereira about this. I am not too sure myself